docker-for-mac/faqs.md

```diff
@@ -220,130 +131,40 @@ export DOCKER_HOST=unix:///var/run/docker.sock
+For information on adding client certificates, see
+[Add client certificates](/docker-for-mac/index/#add-client-certificates) in the Getting Started topic.
 
 ### Can I pass through a USB device to a container?
 
-Unfortunately it is not possible to pass through a USB device (or a
-serial port) to a container. For use cases requiring this, we
-recommend the use of [Docker Toolbox](/toolbox/overview.md).
-
-## Disk Usage
-
-### What is the disk image?
-
-The containers and images are stored in a _disk image_ named `Docker.raw` or
-`Docker.qcow2` depending on your settings (see below).  By default, the disk
-image is stored in `~/Library/Containers/com.docker.docker/Data/vms/0`.
-
-### Qcow2 or Raw?
-
-Starting with High Sierra with Apple Filesystem (APFS) enabled, Docker
-uses disk images in the "raw" format (`Docker.raw`), otherwise in the
-Qcow2 format (`Docker.qcow2`).
-
-### Docker.raw consumes an insane amount of disk space!
-
-This is an illusion.  Docker uses the raw format on Macs running the Apple
-Filesystem (APFS).  APFS supports _sparse files_, which compress long runs of
-zeroes representing unused space.  The output of `ls` is misleading, because it
-lists the logical size of the file rather than its physical size. To see the
-physical size, add the `-ks` switch; to see the logical size in human readable
-form, add `-lh`:
-
-```bash
-$ cd ~/Library/Containers/com.docker.docker/Data/vms/0
-$ ls -klsh Docker.raw
-2333548 -rw-r--r--@ 1 akim  staff    64G Dec 13 17:42 Docker.raw
-```
-
-In this listing, the logical size is 64GB, but the physical size is
-only 2.3GB.
-
-Alternatively, you may use `du` (disk usage):
-
-```bash
-$ du -h Docker.raw
-2,2G	Docker.raw
-```
-
-### How do I reduce the size of Docker.qcow2?
-
-If your Docker for Mac uses the Qcow format, the [disk image
-file](#what-is-the-disk-image) is `Docker.qcow2`.  This file grows
-on-demand up to a default maximum file size of 64GiB.
-
-In Docker 1.12 the only way to free space on the host is to delete
-this file and restart the app. Unfortunately this removes all images
-and containers.
-
-In Docker 1.13 there is preliminary support for "TRIM" to non-destructively
-free space on the host. First free space within the `Docker.qcow2` by
-removing unneeded containers and images with the following commands:
-
-- `docker ps -a`: list all containers
-- `docker image ls`: list all images
-- `docker system prune`: (new in 1.13): deletes all stopped containers, all
-  volumes not used by at least one container, and all images without at least one
-  referring container.
-
-Note the `Docker.qcow2` does not shrink in size immediately.
-In 1.13 a background `cron` job runs `fstrim` every 15 minutes.
-If the space needs to be reclaimed sooner, run this command:
-
-```bash
-$ docker run --rm -it --privileged --pid=host walkerlee/nsenter -t 1 -m -u -i -n fstrim /var
-```
+Unfortunately, it is not possible to pass through a USB device (or a
+serial port) to a container.
```

Comment:
Should we state why this is not possible or what the alternate/preferred way is?

Reply:
Good point. Check with Mathieu on this. I'll update when we have any update.

Reply:
Updated to `Unfortunately, it is not possible to pass through a USB device (or a serial port) to a container as it requires support at the hypervisor level.`